src/styles/sidebar/components/annotation-header.scss

```diff
@@ -10,6 +10,7 @@
 
   &__row {
     display: flex;
+    flex-wrap: wrap-reverse;
```

Comment:
This is in progress, but I need to address the situation that happens when a header (user display name plus edited timestamp plus timestamp) is too wide for its available space, which can happen 5-6 nest-levels deep sometimes. This is step one; it makes wrapping _better_ but not great yet.